tasks/main.yml

```diff
@@ -201,6 +201,29 @@
   with_items: "{{ odoo_role_odoo_dbs }}"
   notify: restart odoo
 
+- name: Check if unaccent extension has been installed ("0" false, "1" true)
+  become: yes
+  become_user: "{{ odoo_role_odoo_user }}"
+  shell: >
+    psql {{ item }} -tAc
+    "SELECT COUNT(*)
+    FROM pg_extension
+    WHERE extname='unaccent';"
+  register: unaccent_installed
+  with_items: "{{ odoo_role_odoo_dbs }}"
+  changed_when: false
+
+- name: "Install unaccent extension: {{ odoo_role_odoo_dbs }}"
+  become: yes
+  become_user: postgres
+  command: >
+    psql {{ item }} -tAc
+    "CREATE EXTENSION \"unaccent\""
+  when: unaccent_installed.results[index].stdout == "0"
+  notify: restart odoo
+  with_items: "{{ odoo_role_odoo_dbs }}"
+  loop_control:
+    index_var: index
```

Comment:
I'm just wondering if the extension installation is actually `odoo-role`'s responsibility or we should add it to `odoo-provisioning`.

The Ansible role still [doesn't support it](https://github.com/geerlingguy/ansible-role-postgresql/pull/137), but maybe we can add it here: https://gitlab.com/coopdevs/odoo-provisioning/-/blob/master/roles/postgresql-odoo/tasks/main.yml

What do you think?

Reply:
I think it's odoo_role responsability because we add the extension to every Odoo database. The extension is already in the system because `geerlingguy.postgresql` installs `postgresql-contrib` Ubuntu package that contains it.

Reply:
Yes, but it's `odoo-provisining` who actually creates all the DBs, right?
https://gitlab.com/coopdevs/odoo-provisioning/-/blob/master/roles/postgresql-odoo/tasks/main.yml#L18